src/containers/pages/ArticlesPage/ArticlesPage.js

```diff
@@ -148,27 +156,37 @@ export class ArticlesPage extends Component {
   }
 
   render() {
+    // TODO(rasendubi): leverage caching
```

Comment:
My name there does not mean that's my personal TODO item. That means I'm the person who could be consulted for clarifications.